packages/editor/src/components/editor/properties/Properties.tsx

```diff
@@ -1,45 +1,73 @@
-import { isNotHiddenField, type Config } from '../../../types/config';
+import { isNotHiddenField, type Config, type Fields } from '../../../types/config';
 import { useData } from '../../../context/useData';
 import { PropertyItem } from './PropertyItem';
-import { Flex, SidebarHeader } from '@axonivy/ui-components';
+import { Accordion, AccordionContent, AccordionItem, AccordionTrigger, Flex, SidebarHeader } from '@axonivy/ui-components';
 import { IvyIcons } from '@axonivy/ui-icons';
-import type { PrimitiveValue } from '@axonivy/form-editor-protocol';
+import { isFreeLayout, type ConfigData, type PrimitiveValue, type ComponentData } from '@axonivy/form-editor-protocol';
+import { groupBy } from '../../../utils/array';
 
 type PropertiesProps = {
   config: Config;
 };
 
+type Entries<T> = {
+  [K in keyof T]: [K, T[K]];
+}[keyof T][];
+
```

Comment:
:astonished:

Reply:
This is because the Object.entries() will change to key type to a string instead of use what is given... 😮‍💨

Reply:
heavy stuff; even for  a java programmer used to generics ;-)
but never mind, every language has its areas where it gets complex